tests/53groups/02read.pl

```diff
@@ -0,0 +1,192 @@
+my $local_viewer_fixture = local_user_fixture( with_events => 0 );
+my $remote_viewer_fixture = remote_user_fixture( with_events => 0 );
+
+foreach my $viewer_fixture ( $local_viewer_fixture, $remote_viewer_fixture) {
+   my $test_name = $viewer_fixture == $local_viewer_fixture ? "local" : "remote";
+
+   test "Get $test_name group profile",
+      requires => [ local_admin_fixture( with_events => 0 ), $viewer_fixture ],
+
+      do => sub {
+         my ( $user, $viewer ) = @_;
+
+         matrix_create_group( $user,
+            name              => "Random Group",
+            avatar_url        => "mxc://example.org/foooooo",
+            short_description => "A random topic for a random group",
+            long_description  => "A longer desc\n\n for a random group",
+         )->then( sub {
+            my ( $group_id ) = @_;
+
+            matrix_get_group_profile( $viewer, $group_id );
+         })->then( sub {
+            my ( $body ) = @_;
+
+            assert_json_keys( $body, qw( name avatar_url short_description long_description ) );
+
+            assert_eq( $body->{name}, "Random Group" );
+            assert_eq( $body->{avatar_url}, "mxc://example.org/foooooo" );
+            assert_eq( $body->{short_description}, "A random topic for a random group" );
+            assert_eq( $body->{long_description}, "A longer desc\n\n for a random group" );
+
+            Future->done( 1 );
+         });
+      };
+
+   test "Get $test_name group users",
+      requires => [ local_admin_fixture( with_events => 0 ), $viewer_fixture ],
+
+      do => sub {
+         my ( $user, $viewer ) = @_;
+
+         matrix_create_group( $user )
+         ->then( sub {
+            my ( $group_id ) = @_;
+
+            matrix_get_group_users( $viewer, $group_id );
+         })->then( sub {
+            my ( $body ) = @_;
+
+            assert_json_keys( $body, qw( chunk ) );
+
+            Future->done( 1 );
+         });
+      };
+
+   test "Add/remove $test_name group rooms",
```

Comment:
this feels like it's duplicating the add/remove tests in 01. should we just get rid of the ones in 01?